graylog2-server/src/main/java/org/graylog2/bundles/BundleImporter.java

```diff
@@ -722,5 +752,15 @@ private void createLookupDataAdapters(String bundleId, Set<LookupDataAdapterBund
                     .build());
             createdLookupDataAdapters.put(dto.id(), dto);
         }
+
+        if (!createdLookupDataAdapters.isEmpty()) {
+            clusterBus.post(DataAdaptersUpdated.create(createdLookupDataAdapters.keySet()));
+        }
+
+        final Collection<LookupDataAdapter> dataAdapters = lookupTableService.getDataAdapters(createdLookupDataAdapters.keySet());
+        final CountDownLatch latch = new CountDownLatch(dataAdapters.size());
+        dataAdapters.forEach(da -> da.addListener(new LatchUpdaterListener(latch), scheduler));
+
+        latch.await(30, TimeUnit.SECONDS);
```

Comment:
We should at least log a short message on either INFO or WARN if the timeout expired (which means that not all caches were refreshed).

Reply:
✅